src/constants.js

```diff
@@ -25,12 +25,18 @@ export const DASHBOARD_BLOCKS_COUNT = 6;
 export const DASHBOARD_TX_COUNT = 6;
 
 export const GENESIS_BLOCK = [
```

Comment:
I don't think we should mix hashes from mainnet and testnet. Maybe we should have two config files: one for the mainnet and another for testnet. @giovanecosta @pedroferreira1 What do you think?

Reply:
Maybe we can start with two different variables in the same constants file? The lib works like this nowadays and we still need to implement this issue (https://github.com/HathorNetwork/hathor-wallet-lib/issues/161) which discusses exactly having a configuration file and being able to easily customize it.

To be honest, the best approach would be for the explorer to get this information (genesis hashes) from the lib in the future but I feel we can start with this solution for now.

Reply:
Ok. We can split into two different variables (one for the testnet and another one for the mainnet).

Reply:
I was looking into the code trying to understand if we could use environment variables to do it, so, we can keep one constant and the explorer agnostic from which network it is connecting. But i realized that we deploy it on Cloudfront, so, no env vars :sweat_smile:

In this case, i agree with two different variables and i'll make a function into `utils.helpers` that checks the network and return if the tx is genesis or not

Reply:
The site is static so there should be a way to inject code based on envvars in compile time.